`littler` installation fails in these days

Since January 16, `littler` seems to be failing to install on `rocker/r-ver:devel`

```log
 #18 9.975 using C compiler: ‘gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0’
#18 10.35 gcc -o r -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g -I/usr/local/lib/R/include littler.c -L/usr/local/lib -Wl,--export-dynamic -fopenmp -L/usr/local/lib -L/usr/local/lib/R/lib -lR -lpcre2-8 -ldeflate -lzstd -llzma -lbz2 -lz -lrt -ldl -lm -licuuc -licui18n -lblas -llapack -Wl,-rpath,/usr/local/lib/R/lib -Wl,-rpath,/usr/local/lib -Wl,-rpath,/usr/lib/x86_64-linux-gnu -Wl,-rpath,/usr/lib/jvm/java-21-openjdk-amd64/lib/server -Wl,-rpath,/usr/local/lib/R/lib -Wl,-rpath,/usr/local/lib -Wl,-rpath,/usr/lib/x86_64-linux-gnu -Wl,-rpath,/usr/lib/jvm/java-21-openjdk-amd64/lib/server -Wl,-rpath,/usr/local/lib/R/lib -Wl,-rpath,/usr/local/lib -Wl,-rpath,/usr/lib/x86_64-linux-gnu -Wl,-rpath,/usr/lib/jvm/java-21-openjdk-amd64/lib/server -Wl,-rpath,/usr/local/lib/R/lib -Wl,-rpath,/usr/local/lib -Wl,-rpath,/usr/lib/x86_64-linux-gnu -Wl,-rpath,/usr/lib/jvm/java-21-openjdk-amd64/lib/server 
#18 10.53 /usr/bin/ld: cannot find -lzstd: No such file or directory
#18 10.53 collect2: error: ld returned 1 exit status
#18 10.53 make: *** [Makevars:31: r] Error 1
#18 10.53 ERROR: compilation failed for package ‘littler’
```

https://github.com/rocker-org/rocker-versioned2/actions/runs/12799426223

@eddelbuettel Any thoughts?

--- a/.github/workflows/r-build-test.yml
+++ b/.github/workflows/r-build-test.yml
@@ -56,7 +56,7 @@ jobs:
       - name: Upload artifacts
         uses: actions/upload-artifact@v4
         with:
-          name: tmp-${{ env.TAG }}
+          name: tmp-${{ matrix.tag }}
           path: tmp
 
   regression-test:

--- a/scripts/setup_R.sh
+++ b/scripts/setup_R.sh
@@ -58,6 +58,7 @@ if [ ! -x "$(command -v r)" ]; then
         liblzma-dev \
         libbz2-dev \
         zlib1g-dev \
+        libzstd-dev \
         libicu-dev"
 
     if [ "$(find /var/lib/apt/lists/* | wc -l)" = "0" ]; then

